list2/ex3/index.js

```diff
@@ -0,0 +1,24 @@
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+
+const DISTANCE_TO_THE_DISCOUNT = 200;
+const PRICE_WITH_DISCOUNT = 0.45;
+const PRICE_WITHOUT_DISCOUNT = 0.50;
+
+export function travelCostCalculation() {
+    console.log('How many km will you travel? (use an INTEGER number)');
+    const distance = parseInt(prompt('>'));
+
+    function calculation() {
+        const moreThan200kmCalculation = distance * PRICE_WITH_DISCOUNT;
+        const lessThan200kmCalculation = distance * PRICE_WITHOUT_DISCOUNT;
+
+        if(distance > DISTANCE_TO_THE_DISCOUNT) {
+             console.log(`Your trip will cost $${moreThan200kmCalculation} dollars.`);
+        } else {
+            console.log(`Your trip will cost $${lessThan200kmCalculation} dollars.`)
+        }
+    }
+    calculation();
```

Comment:
I see almost all of your solutions are following this pattern. It is not very common to have a function defined inside another function. You have 2 main options here;

1. Remove the function
2. Declare the `calculation` function outside the `travelCostCalculation` function and receive the distance as a parameter. 

That rule can be followed in all your exercises. Therefore, even though I don't comment it in other exercises take this into consideration.

Reply:
The function `travelCostCalculation` its not a "real function", the function just exist to export to my menu. And all codes have to have this kind of “main” function.

Reply:
Not sure I got it. Probably your menu is not working as it should since you are declaring and calling the function right after. Therefore, it is the same as not having it.

Reply:
Additionally, the fact that you have a menu and that you are exporting this function should not really change the way you implement it. Therefore, I still think that Declaring the calculation function outside the travelCostCalculation function and receive the distance as a parameter is better.

Reply:
now I understand what you mean, literally useless functions.